app/styles/_core.less

```diff
@@ -53,6 +53,15 @@
       margin-top: 0;
     }
   }
+  .section-header{
```

Comment:
would prefer not to re-do all these margins. Its still the page header so i would still expect it to have the page-header class.  Are the only differences dropping the bottom border and adding(changing?) the bottom padding?